src/components/Dashboard/TodoList.vue

```diff
@@ -756,11 +756,6 @@ export default class TodoList extends Mixins(AllowableActionsMixin, DateMixin) {
   @Getter(useBusinessStore) isDisableNonBenCorps!: boolean
   // @Getter(useAuthenticationStore) isRoleStaff!: boolean
 
-  @Action(useRootStore) setARFilingYear!: (x: number) => void
-  @Action(useRootStore) setArMinDate!: (x: string) => void
-  @Action(useRootStore) setArMaxDate!: (x: string) => void
-  @Action(useRootStore) setNextARDate!: (x: string) => void
```

Comment:
These are no longer set from Todo List, and they are no longer properties in the todo items. Instead, the AR Filing Year is passed as a route query parameter and the others are taken from the business record (store).